pages/wg/outreach.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Outreach Working Group
+subtitle:
+permalink: /wg/outreach/
+---
+
+# Overview
+
+The US-RSE Outreach Working Group is dedicated to broadening the US-RSE's
+engagement with the RSE and wider computational science and engineering (CSE)
+communities. We strive to raise awareness of the US-RSE as an association and
+RSE as a professional designation through engagement with industry, academia,
+national labs, and students.
+
+Those interested in contributing to this goal are encouraged to join our
+bi-weekly virtual meetings. To get involved, visit the
+[`#outreach`](https://usrse.slack.com/messages/outreach) channel on the US-RSE
+slack or contact the
+<a href="mailto:outreach_wg@us-rse.org">Outreach working group list</a>.
+
+# Work Products
+
+The US-RSE Outreach working group is new as of 2022. The following events were
+either completed before the creation of the official working group or are
+initiatives for future work.
+
+## Existing Efforts
+
+### RSE Stories
+
+Created by Vanessa Sochat, the [RSE Stories podcast](https://us-rse.org/rse-stories/)
+aims to cast a spotlight on the differing backgrounds and paths professionals
+have taken to become RSEs and help shape our understanding of what an RSE really is.
+
+The podcast has existed since 2019 and is looking for more volunteers both for
+interviewees and production assistance.
```

Comment:
```suggestion
interviewees and hosts.
```
I wouldn't feel right asking for production assistance - e.g., if someone records an episode, I would offer to help, but I'm not going to ask for help editing mine, if that makes sense!

Reply:
Yup, a-okay with me!